doc/administrator/administrate.rst

```diff
@@ -21,13 +21,13 @@ you should define two ``OGC servers``. Attributes:
 
 * ``Name``: the name of the OGC Server.
 * ``Description``: a description.
-* ``URL``: the server URL, empty to use the internal MapServer.
-* ``WFS support``: whether WFS is supported by the server.
+* ``Basis URL``: the server URL.
 * ``WFS URL``: the WFS server URL, empty to use the same as the ``URL``.
-* ``Type``: the server type used to know which custom attribute will be used.
+* ``Server type``: the server type used to know which custom attribute will be used.
 * ``Image type``: the MIME type of the images (e.g.: ``image/png``).
-* ``Single tile``: whether to use the single tile mode (For future use).
-* ``Auth``: the kind of authentication to use (For future use).
+* ``Authentication type``: the kind of authentication to use.
+* ``WFS support``: whether WFS is supported by the server.
+* ``Is single tile``: whether to use the single tile mode (For future use).
```

Comment:
Not related to this PR, but it is a bit inconsistent to use the naming "WFS support" on the one hand and "Is single tile" on the other hand for boolean values